web/src/games/checkers/index.ts

```diff
@@ -10,7 +10,7 @@ export const checkersGameDef: IGameDef = {
   modes: [{ mode: GameMode.AI }, { mode: GameMode.OnlineFriend }, { mode: GameMode.LocalFriend }],
   minPlayers: 2,
   maxPlayers: 2,
-  description: 'Classic game of Checkers',
+  description: 'a Classic Game',
```

Comment:
Use capital A?

This is also shown on the bottom of the game card, it will be really weird without a capital letter

Reply:
Our CSS styling for the game cards will make it all upper case.  The casing matters for Google search, though:

![image](https://user-images.githubusercontent.com/304383/81253285-3577ec80-8ff6-11ea-8875-f816824f1517.png)